tasks/utils.py

```diff
@@ -136,6 +136,18 @@ def get_version_ldflags(ctx, prefix=None):
     ldflags += "-X {}/pkg/serializer.AgentPayloadVersion={} ".format(REPO_PATH, payload_v)
     return ldflags
 
+def get_git_commit():
```

Comment:
Maybe use this new function from `get_version_ldflags` above, so there is a bit less of duplication.

Reply:
Feel free to ignore this or change it in the next PR so we don't block the merge even longer :)

Reply:
👍